.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   lint:
     docker:
-      - image: circleci/node:8-browsers
+      - image: circleci/node:10-browsers
```

Comment:
May be worth updating the commit message to make it clear that it includes this as well.